src/lib/storage/frame_of_reference/frame_of_reference_iterable.hpp

```diff
@@ -0,0 +1,142 @@
+#pragma once
+
+#include <type_traits>
+
+#include "storage/column_iterables.hpp"
+
+#include "storage/frame_of_reference_column.hpp"
+#include "storage/vector_compression/resolve_compressed_vector_type.hpp"
+
+namespace opossum {
+
+template <typename T>
+class FrameOfReferenceIterable : public PointAccessibleColumnIterable<FrameOfReferenceIterable<T>> {
+ public:
+  explicit FrameOfReferenceIterable(const FrameOfReferenceColumn<T>& column) : _column{column} {}
+
+  template <typename Functor>
+  void _on_with_iterators(const Functor& functor) const {
+    resolve_compressed_vector_type(*_column.offset_values(), [&](const auto& offset_values) {
+      using ZsIteratorType = decltype(offset_values.cbegin());
```

Comment:
We are only talking about „vector compression“, not „zero suppression“, aren‘t we? You‘re using „zs“ in some other places as well.

Reply:
Oops. I’ll change it.